src/GitHub.jl

```diff
@@ -70,6 +70,11 @@ function get_urls(repo::GitHub, page, cdn_url_converter=get_cdn_url_converter(pa
     urls
 end
 
+function get_checksums(repo::GitHub, page)
```

Comment:
Rather than adding this code to every generator that doesn't have a way to get checksums,

add a fall back default method to the parent type of all Repos:
`DataRepo`

which would look like 

```
function get_checksums(repo::DataRepo, page)
     checksums = nothing #Nothing is the correct value to return, as it will trigger gr
     checksums
end
```

Or in short form
```
 get_checksums(repo::DataRepo, page) = nothing
```

That should go in the top level file